python/cudf/cudf/core/frame.py

```diff
@@ -181,6 +181,18 @@ def deserialize(cls, header, frames):
         )
         return cls._from_data(col_accessor)
 
+    @_performance_tracking
+    def nans_to_nulls(self):
```

Comment:
Could you make `IndexedFrame.nans_to_nulls` call `super().nans_to_nulls()` now?

Reply:
done.